app/models/store.rb

```diff
@@ -3,6 +3,7 @@
 class Store < ApplicationRecord
   belongs_to :address, dependent: :destroy
   has_many :users, dependent: :restrict_with_error
+  has_many :vehicles, dependent: :destroy
```

Comment:
```suggestion
  has_many :vehicles, dependent: :restrict_with_error
```

nao queremos apagar uma store se tem vehicles cadastrados nela

Reply:
nao esquece de colocar a traducao de veiculos dentro da store